lib/populator.js

```diff
@@ -24,6 +24,16 @@ function Populator(options) {
   });
 }
 
+// Only adds the dhis version to DHIS URL if it is specified. This ensures backwards compatibility.
+// Needed as DHIS v2.29 api has breaking changes so we need to use the older apis
+function createDhisUrl(dhisVersion, dhisPath) {
+  let tempPath = dhisPath
```

Comment:
For consistency this should be `var` instead of `let`. This can be refactored as a whole later on before we release a new breaking version of the module which drops support for older versions of node.